src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -220,79 +228,166 @@ export const SelectRegistrationTypeField = () => {
             </IconButton>
           )}
         </Box>
-        <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', justifyContent: 'end', mb: '1rem' }}>
-          <FilterVintageIcon
-            color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
+        <Paper
+          elevation={5}
+          sx={{
+            padding: '0.5rem',
+            bgcolor: 'secondary.main',
+            display: 'grid',
+            gap: '0.5rem',
+            gridTemplateAreas: {
+              xs: "'search-bar' 'publication-points-text'",
+              md: "'search-bar publication-points-text'",
+            },
+            gridTemplateColumns: { xs: '1fr', md: '1fr 1fr' },
+            mb: '1rem',
+          }}>
+          <TextField
+            data-testid={dataTestId.registrationWizard.resourceType.resourceTypeSearchField}
+            sx={{ bgcolor: 'white', maxWidth: '15rem', gridArea: 'search-bar' }}
+            placeholder={t('registration.resource_type.search_for_resource_type')}
+            type="search"
+            variant="outlined"
+            label={t('common.search')}
+            InputProps={{
+              startAdornment: <SearchIcon sx={{ mr: '1rem' }} />,
```

Comment:
Tenker vi kan "jukse" litt her også, for å få det likt de fleste feltene som er i wizard i dag, heller enn søkefeltene på fremsiden. Ikke  så viktig, men kanskje bumpe størrelsen bittelitt også, slik at det blir plass til de langste typene også? 
![bilde](https://user-images.githubusercontent.com/6313468/220300650-c0019d31-1d80-48d2-9b6f-f05ac9e10f85.png)


```suggestion
            sx={{ width: '17rem' }}
            type="search"
            variant="filled"
            label={t('common.search')}
            InputProps={{
              endAdornment: <SearchIcon />,
```